feature/dsql/auth/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# v1.0.0 (2024-12-09)
```

Comment:
We need to remove this. When `make release` runs, it'll slurp your .changelog entry and transform it into CHANGELOG.md content (which is expected) - but since you already have the message here it'll be duped:

```
  1 # v1.0.0 (2024-12-13)
  2 
  3 * **Release**: Add Aurora DSQL Auth Token Generator
  4 
  5 # v1.0.0 (2024-12-09)
  6 
  7 * **Release**: Add Aurora DSQL Auth Token Generator
  8 
```